channeldb/payments.go

```diff
@@ -597,28 +597,7 @@ func (d *DB) QueryPayments(query PaymentsQuery) (PaymentsResponse, error) {
 		// literally have to traverse the cursor linearly, which can
 		// take quite a while. So it's an optional query parameter.
 		if query.CountTotal {
```

Comment:
can we update the list payments itest to check this new behavior?

Reply:
Yes, good idea. I'm going to work on it on the following days.